src/main/resources/swagger/thurloe.yaml

```diff
@@ -195,7 +193,6 @@ paths:
           description: Bad Request (e.g. missing or invalid "X-FireCloud-Id" header)
         500:
           description: Internal server error
-          produces: text/plain
```

Comment:
Worth noting: I have a feeling our proxy causes every request to possibly omit an HTML 401 or 503 response.